wiki/Gameplay/Game_modifier/Easy/de.md

```diff
@@ -0,0 +1,78 @@
+
+### osu!
+
+In [osu!](/wiki/Game_mode/osu!) verringert die Mod Easy die [Circle-Size (CS)](/wiki/Client/Beatmap_editor/Song_Setup#circle-size), die [Approach-Rate (AR)](/wiki/Beatmapping/Approach_rate), die [allgemeine Schwierigkeit (OD)](/wiki/Client/Beatmap_editor/Song_Setup#overall-difficulty) und die [HP-Drain](/wiki/Client/Beatmap_editor/Song_Setup#hp-drain-rate) um die Hälfte.
+
+![EZ Gameplay in osu!](img/EZ-comparison-osu.jpg "Vergleich in osu! zwischen einem regulären Spiel (links) und einem Spiel, bei dem die Mod Easy aktiviert ist (rechts)")
+
+Allerdings ist es erwähnenswert, dass viele Spieler die Verwendung der Mod Easy nicht als hilfreich empfinden, um den relativen Schwierigkeitsgrad von Beatmaps zu verringern, insbesondere welche mit hoher Intensität. Das Argument hierfür lautet, dass die verringerte Approach-Rate eine unübersichtliche, schwer zu lesende Situation schafft, bei der die tatsächliche Geschwindigkeit der Beatmap visuell nicht sehr gut widergespiegelt wird (oben abgebildet).
+
+*Anmerkung: Die oben genannten Punkte werden in der Feature-Anfrage "[Let's talk Easy Mod](https://osu.ppy.sh/community/forums/topics/56606)" näher ausgeführt, um die Mod zu verbessern.*
+
+### osu!taiko
+
+In [osu!taiko](/wiki/Game_mode/osu!taiko) verringert die Mod Easy die [Slider-Geschwindigkeit](/wiki/Hit_object/Slider_velocity), die allgemeine Schwierigkeit (OD) und die HP-Drain um die Hälfte.
+
+*Anmerkung: Die Menge der Treffer, die benötigt werden, um einen [Spinner](/wiki/Hit_object/Spinner) zu vervollständigen, wird auf den Wert der OD reduziert.*
+
+Im Gegensatz zu anderen Spielmodi reduziert die Verwendung der Mod Easy in osu!taiko die erforderliche Punktzahl, um Gesundheit zu generieren. Dadurch füllt sich die Lebensleiste viel schneller anstatt, dass zwei zusätzliche "Leben" hinzugefügt werden, da es in osu!taiko unmöglich ist, in der Mitte eines Songs zu scheitern.
+
+### osu!catch
+
+In [osu!catch](/wiki/Game_mode/osu!catch) sind die Effekte die gleichen wie in osu!, mit ein paar Parallelen und Anpassungen: alle Fruits fallen langsamer (erhöhte AR), die Kamera ist "herangezoomt" (erhöhte CS) und zwei zusätzliche "Leben" werden bereitgestellt.
+
+Obwohl nur eine Achse verwendet wird, können die Fruits durch ihre erhöhte Größe und die langsamere Fallrate miteinander verklumpen, wodurch die gleichen gemeinsamen Probleme in der Schwierigkeit wie in osu! entstehen.
+
+![EZ Gameplay catch](img/EZ-comparison-catch.jpg "Vergleich in osu!catch zwischen einem regulären Spiel (links) und einem Spiel, bei dem die Mod Easy aktiviert ist (rechts)")
+
+### osu!mania
+
+In [osu!mania](/wiki/Game_mode/osu!mania) sind die Effekte die gleichen wie in osu!taiko, außer, dass dem Spieler die zwei zusätzlichen Leben *tatsächlich* bereitgestellt werden und dass die Slider-Geschwindigkeit sowie die standardmäßige Tastenanzahl dasselbe sein werden.
```

Comment:
```suggestion
In [osu!mania](/wiki/Game_mode/osu!mania) sind die Effekte die gleichen wie in osu!taiko, außer, dass dem Spieler die zwei zusätzlichen Leben *tatsächlich* bereitgestellt werden und dass die Slider-Geschwindigkeit sowie die standardmäßige Tastenanzahl dieselbe sein werden.
```

Reply:
I wasn't sure which was correct here really. Now that you mention it, is `dieselben` correct, due to the plural?